### PR TITLE
:pencil: Update README.md

### PR DESCRIPTION
**Pull request by teners**
Solves #4.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,8 +2,75 @@
...
+poetry install
+```
+
+### Run
+
+For development purposes, you can skip the Docker part to not to bother with all
+those fancy production accoutrements, and just use the run script:
+
+```shell
+sh run-app.sh [WORKERS_NUM PORT]
```
Comment (b0g3r) [inline README.md]:
```suggestion
sh run-app.sh [WORKERS_NUM] [PORT]
```

File: README.md [hunk 2]
```diff
@@ -2,8 +2,75 @@
...
+so, beware.
+
+If you want to reduce the overhead even more you can run the app with uvicorn:
+
+```shell
+uvicorn c4p2n.api:app
+```
+
+Now the app is up and running and you can find the OpenAPI schema at
+https://localhost:8000.
```
Comment (b0g3r) [inline README.md]:
```suggestion
http://localhost:8000/redoc for redoc and http://localhost:8000/docs for swagger.
```

Comment (teners) [reply README.md]: Ouch. Should it be `/docs` or `/redoc`, actually?

Comment (b0g3r) [reply README.md]: /docs for swagger, /redoc for redoc

Comment (b0g3r) [reply README.md]: Friendly reminder

File: README.md [hunk 3]
```diff
@@ -2,8 +2,75 @@
...
+To install dependencies from the project root run:
+
+```shell
+poetry install
+```
+
+### Run
+
+For development purposes, you can skip the Docker part to not to bother with all
+those fancy production accoutrements, and just use the run script:
```
Comment (ave2me) [inline README.md:51]: much native :dog:

Comment (teners) [reply README.md:51]: Felt cute, might rewrite later ✨

**Review by ave2me — APPROVED** [review]

**Pull request merged**